source/transactions-convenient-api/tests/commit-writeconcernerror.yml

```diff
@@ -0,0 +1,176 @@
+database_name: &database_name "withTransaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  -
+    description: commitTransaction is not retried after write concern error (WriteConcernFailed)
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 1 }
+      data:
+          failCommands: ["commitTransaction"]
+          closeConnection: false
```

Comment:
Sorry, thought I had already commented on this, but I think I only mentioned it in Slack: 

None of the failpoints in this file work unless the `closeConnection : false` is removed.  With it, the writeConcernError is not included in the response.

Please remove all the closeConnection fields (as false is the default anyway).

Reply:
Removed in https://github.com/mongodb/specifications/pull/435/commits/cb72ce2b81408ba345f2f4006948ff697c32aad9